tests/tests.js

```diff
@@ -171,6 +175,31 @@ describe('Pricing', () => {
       expect(formatPriceSpy).to.have.callCount(1)
     })
 
+    it('returns the price of commuter products (parking)', () => {
+      const selectedOptions = {
+        benefit: 'parking'
+      }
+
+      const price = pricing.calculateProductPrice(products.commuter, employee, selectedOptions)
+
+      expect(price).to.equal(175)
+      expect(calculateCommuterPriceSpy).to.have.callCount(1)
+      expect(getEmployerContributionSpy).to.have.callCount(1)
+      expect(formatPriceSpy).to.have.callCount(1)
+    })
+    it('returns the price of commuter products (train)', () => {
+      const selectedOptions = {
+        benefit: 'train'
+      }
+      const price = pricing.calculateProductPrice(products.commuter, employee, selectedOptions)
+
+      expect(price).to.equal(9.75)
+      expect(calculateCommuterPriceSpy).to.have.callCount(1)
+      expect(getEmployerContributionSpy).to.have.callCount(1)
+      expect(formatPriceSpy).to.have.callCount(1)
+    })
```

Comment:
These tests should have been in your last commit. The directions were for tests to be committed first so we can show the TDD flow.